Dropbox/Dropbox.download.recipe

```diff
@@ -3,13 +3,22 @@
 <plist version="1.0">
 <dict>
     <key>Description</key>
-    <string>Downloads the current release version of Dropbox</string>
+    <string>Downloads the current release version of Dropbox
+
+ARCHITECTURE
+- x86_64 (default)
+- arm64
+</string>
     <key>Identifier</key>
     <string>com.github.autopkg.download.dropbox</string>
     <key>Input</key>
     <dict>
         <key>NAME</key>
         <string>Dropbox</string>
+        <key>ARCHITECTURE</key>
+        <string>x86_64</string>
+        <key>DOWNLOAD_URL</key>
+        <string>https://www.dropbox.com/download?plat=mac&amp;arch=%ARCHITECTURE%&amp;full=1</string>
```

Comment:
Why does DOWNLOAD_URL need to be an input variable? I could see a rationale for adding a PLATFORM variable, but not the whole URL.